update/delta/delta_1.3.6_add_vl_for_SIA_export.sql

```diff
@@ -135,6 +135,9 @@ UPDATE qwat_vl.precisionalti SET sia405code = 'genau' WHERE value_fr = 'Inférie
 UPDATE qwat_vl.precisionalti SET sia405code = 'genau' WHERE value_fr = 'Inférieure à 5 cm';
 
 -- TODO : complete mapping
+UPDATE qwat_vl.pressurecontrol_type SET sia405code = 'Schacht.unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.pressurecontrol_type SET sia405code = 'Schacht.unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.pressurecontrol_type SET sia405code = 'Schacht.unbekannt' WHERE value_fr = 'à déterminer';
 UPDATE qwat_vl.pressurecontrol_type SET sia405code = 'Schacht.Druckbrecher' WHERE value_en = 'reducer';
```

Comment:
Could also be Rohrleitungsteil.Art = Diverse.Druckminderventil (Composant.Genre  = divers.reducteur_de_pression)

Reply:
This is actually a VL only for structures (installations), mapping it to coumpound would be equivalent as moving it to the element_part of QWAT data model.

Maybe we must ensure that element_part goes into compound when exported to SIA